pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, makeDesktopItem, ncurses, libX11, boost, cmake }:
+
+let
+  pname = "tome2";
+  description = "A dungeon crawler similar to Angband, based on the works of Tolkien";
+
+  desktopItem = makeDesktopItem {
+    desktopName = pname;
+    name = pname;
+    exec = "@out@/bin/${pname}-x11";
```

Comment:
```
~/nixpkgs> cat /nix/store/vfmh86njwrm7xd3daili5r4knx163wcd-tome2-2.4/share/applications/tome2.desktop
[Desktop Entry]
Type=Application
Exec=@out@/bin/tome2-x11
Terminal=False
Name=tome2
Categories=Game;RolePlaying;
Icon=tome2
Comment=A dungeon crawler similar to Angband, based on the works of Tolkien
GenericName=tome2
```

Reply:
Changed the @out@ thing to just name of the binary. Or was there something else you meant?